server/apps/acсount/views.py

```diff
@@ -0,0 +1,13 @@
+from .models import User, Team
+from .serializers import UserSerializer, TeamSerializer
+
+from django.shortcuts import get_list_or_404
+
+from rest_framework import viewsets, generics
+from rest_framework.response import Response
+from rest_framework.permissions import IsAuthenticated
+
+
+class UserListView(generics.ListAPIView):
+    serializer_class = UserSerializer
+    permission_classes = (IsAuthenticated)
```

Comment:
The `permission_classes` attribute should be iterable. In your case, `permission_classes` is equal to the `IsAuthenticated` class.
Try converting it to a tuple or a list of classes.

Source: https://www.django-rest-framework.org/api-guide/generic-views/#examples